triggers.md

```diff
@@ -1,3 +1,133 @@
 # Prerendering triggers
 
-TODO Jeremy port from Google doc
+A web-facing "trigger" is a mechanism for web content to permit prerendering of certain URLs. It may include additional daata that informs the user agent about prerendering.
```

Comment:
```suggestion
A web-facing "trigger" is a mechanism for web content to permit prerendering of certain URLs. It may include additional data that informs the user agent about prerendering.
```